lua/jdtls/setup.lua

```diff
@@ -124,7 +124,7 @@ end
 ---@return string
 local function sha1(val)
   local cmd = {
-    "python",
+    vim.fn.executable("python3") and "python3" or "python",
```

Comment:
```suggestion
    vim.fn.executable("python3") == 1 and "python3" or "python",
```

`vim.fn.executable` returns 1 or 0, and 0 is truthy in lua so this would always return `python3`